src/Chat/ChatProvider.tsx

```diff
@@ -159,12 +149,13 @@ export const ChatProvider = React.forwardRef<any, ChatScreenProps>(
 
     useEffect(() => {
       if (conversationInfo?.id) {
+        setLoadEarlier(true);
         FirestoreServicesInstance.countAllMessages().then((total) => {
           totalMessages.current = total;
         });
         FirestoreServicesInstance.getMessageHistory().then((res) => {
           FirestoreServicesInstance.changeReadMessage();
-          setLoadEarlier(true);
+          setLoadEarlier(false);
```

Comment:
`setLoadEarlier(false);` it should inside finally. Because it always true if getMessageHistory is error